### PR TITLE
chore: minor linting on the simple-blog example

### PR DESCRIPTION
**Pull request by JensRoland**
Linting with markdown-linter and normalized some trailing newlines in files

### CODE REVIEW

File: packages/examples/simple-blog/blog/scaleable-design-system.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ Design systems are key to managing code at scale, but creating one that’s flex
 
 1. **Reusable components**: Break UI elements into reusable, abstracted components. A button should be built independently from where it’s used. This promotes consistency across properties and apps.
```
Comment (nobkd) [inline packages/examples/simple-blog/blog/scaleable-design-system.md]:
```suggestion
2. **Reusable components**: Break UI elements into reusable, abstracted components. A button should be built independently from where it’s used. This promotes consistency across properties and apps.
```

File: packages/examples/simple-blog/blog/scaleable-design-system.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ Design systems are key to managing code at scale, but creating one that’s flex
...
 
-3. **Clear naming conventions**: Utilize consistent, semantic naming for classes, variables, mixins, etc. This improves developer efficiency and organization.
-
+1. **Clear naming conventions**: Utilize consistent, semantic naming for classes, variables, mixins, etc. This improves developer efficiency and organization.
```
Comment (nobkd) [inline packages/examples/simple-blog/blog/scaleable-design-system.md]:
```suggestion
3. **Clear naming conventions**: Utilize consistent, semantic naming for classes, variables, mixins, etc. This improves developer efficiency and organization.
```

File: packages/examples/simple-blog/index.md [hunk 3]
```diff
@@ -3,8 +3,8 @@ include: [ gallery, motion ]
 content_collection: blog
 ---
 
-# Minimalist, UX developer, designer, urban explorer.
-I’m Emma Bennett, an user experience developer from Berlin. I build websites that are exceptionally well designed — inside, and outside.
+# Minimalist, UX developer, designer, urban explorer
 
+I’m Emma Bennett, an user experience developer from Berlin. I build websites that are exceptionally well designed — inside, and outside.
```
Comment (nobkd) [inline packages/examples/simple-blog/index.md]:
```suggestion
I'm Emma Bennett, an user experience developer from Berlin. I build websites that are exceptionally well designed — inside, and outside.
```

**Review by nobkd — CHANGES_REQUESTED** [review]

**Comment by nobkd** [comment]
Thanks

File: packages/examples/simple-blog/blog/class-naming-strategies.md [hunk 4]
```diff
@@ -8,17 +8,14 @@ credits: ui8
...
 
-
 ## Keep them short
 
 When designing for the web, keeping CSS in mind from the start prevents frustration down the road. For example, layered graphical elements must be broken down into achievable HTML components. Typography and color schemes are also heavily influenced by CSS possibilities.
 
-
 Designers should have a strong handle on core CSS concepts like the box model, selectors, specificity and inheritance.
 
 > When designing for the web, keeping CSS in mind from the start prevents frustration down the road. For example, **typography and color schemes are  heavily influenced by CSS** and layered graphical elements must be broken down into achievable HTML components.
```
Comment (seneca) [inline packages/examples/simple-blog/blog/class-naming-strategies.md:18]: double space between "are heavily"

Comment (nobkd) [reply packages/examples/simple-blog/blog/class-naming-strategies.md:18]: Thanks. Fixed in https://github.com/nuejs/nue/commit/695111789427a09fe6f877f557159dbd066c1ac7

**Pull request merged**